test/vfd_swmr_zoo_writer.c

```diff
@@ -74,335 +57,437 @@ print_cache_hits(H5C_t H5_ATTR_UNUSED *cache)
 #endif
 
 void
-zoo_create_hook(hid_t fid)
+zoo_create_hook(hid_t H5_ATTR_UNUSED fid)
 {
     dbgf(3, "%s: enter\n", __func__);
     if (writer)
-        H5Fvfd_swmr_end_tick(fid);
+        decisleep(1);
 }
 
 static void
 usage(const char *progname)
 {
-    fprintf(stderr, "usage: %s [-C] [-S] [-W] [-a] [-e] [-m] [-q] [-v]\n", progname);
+    fprintf(stderr, "usage: %s [-C] [-S] [-a] [-e] [-p] [-q] [-v]\n", progname);
     fprintf(stderr, "\n  -C: skip compact dataset tests\n");
-    fprintf(stderr, "  -S: do not use VFD SWMR\n");
-    fprintf(stderr, "  -W: do not wait for SIGINT or SIGUSR1\n");
-    fprintf(stderr, "  -a: run all tests, including variable-length data\n");
-    fprintf(stderr, "  -e: print error stacks\n");
-    fprintf(stderr, "  -m ms: maximum `ms` milliseconds pause between\n");
-    fprintf(stderr, "         each create/delete step\n");
-    fprintf(stderr, "  -q: be quiet: few/no progress messages\n");
-    fprintf(stderr, "  -v: be verbose: most progress messages\n");
+    fprintf(stderr,   "  -S: do not use VFD SWMR\n");
+    fprintf(stderr,   "  -a: run all tests, including variable-length data\n");
+    fprintf(stderr,   "  -e: print error stacks\n");
+    fprintf(stderr,   "  -l secs: maximal seconds for reader to validate the zoo\n");
```

Comment:
This "-l secs: maximal seconds for reader to validate the zoo\n" seems not accurate. From the code it seems that it is "maximal number of ticks for reader to validate the zoo\n"

Reply:
I corrected it.